client/client.go

```diff
@@ -110,3 +123,32 @@ func newClientTimeout(options ClientTimeout) ClientTimeout {
 
 	return ct
 }
+
+// / Generator of unique lease id
```

Comment:
Please remove this useless slash.

Reply:
Oops, sorry I missed that! I've updated it as you recommended.